metrics_layer/cli/seeding.py

```diff
@@ -304,12 +328,71 @@ def make_fields(self, column_data: str):
                 field["type"] = "time"
                 field["timeframes"] = ["raw", "date", "week", "month", "quarter", "year"]
                 field["datatype"] = metrics_layer_type
+            elif metrics_layer_type == "string" and auto_tag_searchable_fields:
+                field["field_type"] = "dimension"
+                field["type"] = "string"
+                searchable_field_candidates.append(row["COLUMN_NAME"])
             else:
                 field["field_type"] = "dimension"
                 field["type"] = metrics_layer_type
             fields.append(field)
+        if searchable_field_candidates:
+            column_cardinalities_query = self.column_cardinalities_query(
+                column_names=searchable_field_candidates, schema_name=schema_name, table_name=table_name
+            )
+            column_cardinalities = self.run_query(query=column_cardinalities_query)
+
+            # Get the column names that have a cardinality of less than 100
+            # Note: running the query doesn't preserve the column name cases
+            searchable_column_names = [
+                col.lower().rsplit("_cardinality", 1)[0]
+                for col in column_cardinalities.columns
+                if column_cardinalities.loc[0, col] < 100
```

Comment:
make this settable with the call to seed (e.g. if I want everything searchable with less than 1000 cardinality, let me set that via a argument to the `seed` function)